R/task.R

```diff
@@ -468,9 +483,10 @@ task_log_value <- function(id, outer = FALSE, root = NULL) {
 ##' @inheritParams logwatch::logwatch
 ##'
 ##' @export
-task_log_watch <- function(id, poll = 1, skip = 0, timeout = Inf,
+task_log_watch <- function(id, follow = TRUE, poll = 1, skip = 0, timeout = Inf,
                            progress = NULL, root = NULL) {
   root <- hipercow_root(root)
+  id <- follow_retry_map(id, root)
```

Comment:
See below - I *think* if (follow) { ... } - but why you would call this on a dead job with follow = FALSE...